README.md

```diff
@@ -6,19 +6,19 @@ Install via NPM:
 
 ```bash
 npm install hansken-js
-
 ```
 
-
+## Usage
 ```javascript
-<script type="text/javascript" src="hansken.js"></script>
-<script type="text/javascript">
-    const hansken = new HanskenClient('https://gatekeeper01.hansken.org/gatekeeper');
+import {HanskenClient} from './src/hansken.js';
 
-    hansken.projects().then((projects) => {
-        projects.forEach((project) => {
-            document.write(`<li>${project.name}`);
-        });
+const hansken = new HanskenClient('/gatekeeper');
```

Comment:
now that I see the hansken client, maybe you can make this an object? like so;
```javascript
const config = {
  gatekeeper: '/gatekeeper',
  preference: '/preference'
}

const hansken = new HanskenClient(config);
```

Reply:
I expect that this constructor will take no more than 3 arguments in the future; gatekeeper, keystore and preference urls. For now, I'd like to keep it this way as it takes less code to initialize. When we need more config parameters, we can add your suggestion in a future release.